### PR TITLE
Fix: Fixed issue where adaptive layout is not disabled when the sync layout setting is on

### LINKED ISSUE #14854: Bug: Adaptive layout is broken
### Description

Starting in v3.2.4, adaptive layout is missing from the right click context menu, and doesn't work when toggled from the layout picker.

### Steps To Reproduce

1. Open layout picker
2. Toggle the option for adaptive
3. Nothing happens

### Requirements

- Correctly apply adaptive layout

### Files Version

v3.2.4

### Windows Version

Windows 11

### Log File

NA

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #14854 

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [X] Did you implement any design changes to an existing feature?
   - [X] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### CODE REVIEW

File: src/Files.App/Strings/en-US/Resources.resw [hunk 1]
```diff
@@ -3740,4 +3740,7 @@
   <data name="Customized" xml:space="preserve">
     <value>Customized</value>
   </data>
+  <data name="AdaptiveLayoutDisabledNotification" xml:space="preserve">
+    <value>Adaptive layout is not available when the sync option is on.</value>
```
Comment (yaira2) [inline src/Files.App/Strings/en-US/Resources.resw:3744]:
```suggestion
    <value>The option for adaptive layout cannot be used while the sync option is enabled.</value>
```

File: src/Files.App/Views/Settings/LayoutPage.xaml [hunk 2]
```diff
@@ -200,5 +210,13 @@
...
 
 		</StackPanel>
+
+		<!--  Adaptive Layout Disabled Teaching Tip  -->
+		<TeachingTip
+			CloseButtonContent="{helpers:ResourceString Name=OK}"
+			IsLightDismissEnabled="True"
+			IsOpen="{x:Bind ViewModel.ShowAdaptiveDisabledTeachingTip, Mode=TwoWay}"
+			Subtitle="{helpers:ResourceString Name=AdaptiveLayoutDisabledNotification}"
+			Target="{x:Bind LayoutTypeComboBox}" />
```
Comment (yaira2) [inline src/Files.App/Views/Settings/LayoutPage.xaml:220]: Do you think it would be better to target the sync preferences setting?

Comment (hishitetsu) [reply src/Files.App/Views/Settings/LayoutPage.xaml]: Yes if the teaching tip message is replaced as you suggest, so I have changed the target.

**Comment by yaira2** [comment]
LGTM

**Pull request merged**